test/unit/ping-centre/PingCentre.test.js

```diff
@@ -340,6 +340,7 @@ describe("PingCentre", () => {
           locale: FAKE_LOCALE,
           topic: "activity-stream",
           client_id: FAKE_TELEMETRY_ID,
+          version: "69.0a1",
```

Comment:
@ncloudioj this change was required because previously `AppConstants.MOZ_APP_VERSION` was `undefined` and the key would be dropped when creating a ping in the tests. Just want to confirm sending `version` is expected (I see it defined in our data_dictionary).

Reply:
Yes, `version` is a required field in every single ping. Thanks for the fix!